devtools/armnotify.py

```diff
@@ -4,23 +4,25 @@
 """
 Automatic-Ripping-Machine Development Tools
     ARM Notify test tools
+
+    TODO: this needs fixing, it can't call arm because it causes looping issues with dependencies
 """
 
-import log
-import sys
-sys.path.insert(0, '/opt/arm')
-from arm.ripper import utils        # noqa E402
-from arm.models.job import Job   # noqa E402
+# import log
+# import sys
+# sys.path.insert(0, '/opt/arm')
+# from arm.ripper import utils        # noqa E402
+# from arm.models.job import Job   # noqa E402
 
 
-def test():
-    """
-    Run a notification
-        INPUT: none
-        OUTPUT: none
-    """
-    try:
-        job = Job("/dev/null")
-        utils.notify(job, "ARM notification", "This is a notification by the ARM-Notification Test!")
-    except Exception as error:
-        log.error(f"ARM Notify failed to run - {error}")
+# def test():
+#     """
+#     Run a notification
+#         INPUT: none
+#         OUTPUT: none
+#     """
+#     try:
+#         job = Job("/dev/null")
+#         utils.notify(job, "ARM notification", "This is a notification by the ARM-Notification Test!")
+#     except Exception as error:
+#         log.error(f"ARM Notify failed to run - {error}")
```

Comment:
Go ahead and strip out the commented out code You can always refer to the git history if you don't want to rewrite it in the future.